governance/default.go

```diff
@@ -770,7 +779,7 @@ func (gov *Governance) UpdateCurrentGovernance(num uint64) {
 	newNumber, newGovernanceSet, _ := gov.ReadGovernance(num)
 
 	// Do the change only when the governance actually changed
-	if newGovernanceSet != nil && newNumber != atomic.LoadUint64(&gov.actualGovernanceBlock) {
+	if newGovernanceSet != nil && newNumber > atomic.LoadUint64(&gov.actualGovernanceBlock) {
```

Comment:
What is the reason of changing this line?

Reply:
@ehnuje UpdateCurrentGovernance() is called by snapshot() when blockNumber % Epoch == 0. So while following the incoming blocks, there was no problem. But if istanbul.getSnapshot() is called for the old Epoch block, though `newNumber < actualGovernanceBlock`, the update can be performed. This change is to prevent that current information is overwritten by the old one.